ui/src/pages/version/components/forms/components/transformer/InputPanel.js

```diff
@@ -0,0 +1,141 @@
+      const items = euiDragDropReorder(inputs, source.index, destination.index);
+      onChangeHandler([...items]);
+    }
+  };
+
+  return (
+    <Panel title="Input" contentWidth="75%">
+      <EuiDragDropContext onDragEnd={onDragEnd}>
+        <EuiFlexGroup direction="column" gutterSize="s">
+          <EuiDroppable droppableId="INPUTS_DROPPABLE_AREA" spacing="m">
+            {inputs.map((input, idx) => (
+              <EuiDraggable
+                key={`${idx}`}
+                index={idx}
+                draggableId={`${idx}`}
+                customDragHandle={true}
+                disableInteractiveElementBlocking>
+                {provided => (
+                  <EuiFlexItem key={`input-${idx}`}>
+                    {input.feast && (
+                      <FeastResourcesContextProvider
+                        project={input.feast[0].project}>
+                        <FeastInputCard
+                          index={idx}
+                          table={input.feast[0]}
+                          tableNameEditable={true}
+                          onChangeHandler={onChange(`${idx}.feast.0`)}
+                          onDelete={
+                            inputs.length > 1 ? onDeleteInput(idx) : undefined
+                          }
+                          dragHandleProps={provided.dragHandleProps}
+                        />
+                      </FeastResourcesContextProvider>
+                    )}
+
+                    {input.tables && (
+                      <TableInputCard
+                        index={idx}
+                        table={input.tables[0]}
+                        onChangeHandler={onChange(`${idx}.tables.0`)}
+                        onColumnChangeHandler={onChange(
+                          `${idx}.tables.0.columns`
+                        )}
+                        onDelete={
+                          inputs.length > 1 ? onDeleteInput(idx) : undefined
+                        }
+                        dragHandleProps={provided.dragHandleProps}
+                      />
+                    )}
+
+                    {input.variables && (
+                      <VariablesInputCard
+                        index={idx}
+                        variables={input.variables}
+                        onChangeHandler={onChange(`${idx}.variables`)}
+                        onDelete={
+                          inputs.length > 1 ? onDeleteInput(idx) : undefined
+                        }
+                        dragHandleProps={provided.dragHandleProps}
+                      />
+                    )}
+
+                    <EuiSpacer size="s" />
+                  </EuiFlexItem>
+                )}
+              </EuiDraggable>
+            ))}
+          </EuiDroppable>
+
+          <EuiFlexItem>
+            <EuiFlexGroup>
+              <EuiFlexItem>
+                <AddButton
+                  title="+ Add Feast Features"
+                  description="Use Feast features as input"
```

Comment:
@pradithya Need help with copywriting here and below (Add Feast Features, Generic Table, and Variables buttons).

Reply:
Add Feast Features: Create a table by using features retrieved from Feast
Generic Table: Create a table from request/response json payload or other inputs
Add Variables: Declare a variable, variable can be used as input to expression